package.json

```diff
@@ -27,14 +27,17 @@
   "dependencies": {
     "@expo/react-native-action-sheet": "4.0.1",
     "@gorhom/bottom-sheet": "^4",
+    "@notifee/react-native": "^7.7.1",
     "@react-native-async-storage/async-storage": "1.18.1",
     "@react-native-community/blur": "^4.3.2",
     "@react-native-community/datetimepicker": "6.7.3",
     "@react-native-community/netinfo": "9.3.7",
+    "@react-native-community/push-notification-ios": "^1.11.0",
```

Comment:
Why do we need this?

Reply:
It's part of `react-native-push-notification`. The app doesn't compile to iOS without it.